### PR TITLE
Github actions creation

### PR DESCRIPTION
**Pull request by martysp21**
[PVCORE-3551](https://issues.redhat.com/browse/PVCORE-3551)
[PVCORE-3552](https://issues.redhat.com/browse/PVCORE-3552)

Creation of GitHub action for sending data through Pelorus webhook

### CODE REVIEW

**Comment by apetryla** [comment]
LGTM, good job!

File: committime_template/action.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: "Commit time action"
+description: "Composite action for Commit time metric for Pelorus."
+
+inputs:
+  app:
+    description: "App to monitor"
+    required: true
+  url:
+    description: "Url of pelorus endpoint"
+    default: http://webhook-exporter-pelorus.apps.grey.dev.iad2.dc.paas.redhat.com/pelorus/webhook
```
Comment (mateusoliveira43) [inline committime_template/action.yml:10]: I would make this field required as well

Comment (mftb) [reply committime_template/action.yml:10]: Fully agree. This is something that needs to be a parameter, not a hardcoded value.

Also don't forget to change this for the other action template.

Comment (xpokor85) [reply committime_template/action.yml]: Good point. Changed

File: README.md [hunk 2]
```diff
@@ -1,2 +1,28 @@
...
+### Example of usage
+
+```yaml
+name: Send-Data
+on: push
+jobs:
+    check-committime-template:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: ./committime_template
```
[the remark this answers is not in the document]
Comment (mftb) [reply README.md:23]: +1 this

[the remark this answers is not in the document]
Comment (mftb) [reply README.md:23]: IMO this can be resolved.

**Comment by mftb** [comment]
LGTM

**Review by mateusoliveira43 — APPROVED** [review]

**Comment by xpokor85** [comment]
@mftb @mateusoliveira43 Could you merge this? I don't have the rights.

**Comment by mateusoliveira43** [comment]
@xpokor85 can you check your email/github inbox for a collaborator invite? that should allow you to merge PRs

**Pull request merged**